src/stage/stage.ts

```diff
@@ -96,8 +96,6 @@ declare global {
  *                               in percent of scene bounding box
  * @property {Float} fogFar - position where the fog is in full effect
  *                              in percent of scene bounding box
- * @property {String} fogMode - how to interpret fogNear/Far values: "scene" for scene-relative, "camera" for camera-relative
```

Comment:
I think this totally makes sense to remove these; I can't imagine anyone wanting clip one way and fog the other. The docs above should be adjusted to note that clipMode/clipScale also affect fog.

Reply:
Thanks, will update docs